### PR TITLE
use dev-scripts on CI

### PR DESCRIPTION
**Pull request by Kriechi**
This should help us keep the install routine clean and tested on all platforms.

### CODE REVIEW

File: dev.sh [hunk 1]
```diff
@@ -2,6 +2,8 @@
 set -e
 VENV=./venv
 
+pip install -U virtualenv || sudo pip install -U virtualenv
```
Comment (mhils) [inline dev.sh]: I don't like that we upgrade system Python things here. Why is that necessary?

Comment (Kriechi) [inline dev.sh]: upgrade is not necessary - but why not?
install is necessary - travis does not ship with virtualenv.

**Comment by Kriechi** [comment]
@mhils please take another look, I fixed the things you mentioned.
